packages/trader/src/Modules/Contract/Components/AccumulatorsStats/__tests__/accumulators-stats.spec.tsx

```diff
@@ -27,54 +31,54 @@ describe('AccumulatorsStats', () => {
     });
     it('should show manual after info icon is clicked', () => {
-        const { container } = render(<AccumulatorsStats />, {
+        render(<AccumulatorsStats />, {
             container: document.body.appendChild(modal_root_el),
         });
-        fireEvent.click(container.querySelector('.info'));
+        fireEvent.click(screen.getByTestId('dt_ic_info_icon'));
         expect(screen.getByTestId('dt_accumulators_stats_manual_video')).toBeInTheDocument();
     });
     it('should render partial history values (tick counters) when initially collapsed in desktop', () => {
         render(<AccumulatorsStats />);
         expect(screen.getAllByTestId('dt_accu_stats_history_counter').length).toEqual(ROW_SIZES.DESKTOP_COLLAPSED);
     });
     it('should render partial history values (tick counters) when initially collapsed in mobile', () => {
-        isMobile.mockReturnValue(true);
-        isDesktop.mockReturnValue(false);
+        (isMobile as jest.Mock).mockReturnValue(true);
+        (isDesktop as jest.Mock).mockReturnValue(false);
         render(<AccumulatorsStats />);
         expect(screen.getAllByTestId('dt_accu_stats_history_counter').length).toEqual(ROW_SIZES.MOBILE_COLLAPSED);
     });
     it('should expand in desktop when accordion_toggle_arrow is clicked', () => {
-        const { container } = render(<AccumulatorsStats />);
+        render(<AccumulatorsStats />);
         expect(screen.getAllByTestId('dt_accu_stats_history_counter').length).toEqual(ROW_SIZES.DESKTOP_COLLAPSED);
 
-        fireEvent.click(container.querySelector('.accordion-toggle-arrow'));
+        fireEvent.click(screen.getByTestId('dt_accordion-toggle-arrow'));
         const row = screen.getAllByTestId('dt_accu_stats_history_row')[0];
         expect(within(row).getAllByTestId('dt_accu_stats_history_counter').length).toEqual(ROW_SIZES.DESKTOP_EXPANDED);
         expect(screen.getAllByTestId('dt_accu_stats_history_counter').length).toEqual(20);
     });
     it('should show MobileDialog with full "Stay in history" in mobile when accordion_toggle_arrow is clicked', () => {
-        isMobile.mockReturnValue(true);
-        isDesktop.mockReturnValue(false);
-        const { container } = render(<AccumulatorsStats />, {
+        (isMobile as jest.Mock).mockReturnValue(true);
+        (isDesktop as jest.Mock).mockReturnValue(false);
+        render(<AccumulatorsStats />, {
             container: document.body.appendChild(modal_root_el),
         });
         expect(screen.getAllByTestId('dt_accu_stats_history_counter').length).toEqual(ROW_SIZES.MOBILE_COLLAPSED);
 
-        fireEvent.click(container.querySelector('.accordion-toggle-arrow'));
-        const mobile_dialog = document.body.querySelector('.dc-mobile-dialog__accumulators-stats');
+        fireEvent.click(screen.getByTestId('dt_accordion-toggle-arrow'));
+        const mobile_dialog = screen.getByTestId('dt_mobile_dialog_stats');
```

Comment:
@henry-deriv let's use existing 'dt_mobile_dialog' that we have in MobileDialog component.🙂